x-pack/platform/plugins/shared/fleet/public/applications/fleet/sections/settings/components/edit_output_flyout/output_form_validators.tsx

```diff
@@ -369,6 +369,34 @@ export function validateDynamicKafkaTopics(value: Array<EuiComboBoxOptionOption<
         })
       );
     }
+
+    const getAllIndices = (str: string, substring: string): number[] => {
+      const indices = [];
+      let indx = str.indexOf(substring);
+      while (indx !== -1) {
```

Comment:
Nit: `indx` is only one letter off from `index` so let's use the full name here.